src/Alert/Alert.js

```diff
@@ -72,6 +72,7 @@ export const Alert = ({ classes = {}, children, ref, ...props }) => {
   const icon = props.icon ?? defaultIcon;
   const DefaultDismiss = (
     <DismissButton text='Dismiss'
+                   srText='alert'
```

Comment:
Add screen reader text for dismiss button so it reads out as "Dismiss alert" instead of just "Dismiss"